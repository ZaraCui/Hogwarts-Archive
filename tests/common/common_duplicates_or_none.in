ADD COLLECTION tests/data/books_small.csv
ADD STUDENT Ron Weasley
COMMON 100000 100000
COMMON 100000 100001
EXIT

