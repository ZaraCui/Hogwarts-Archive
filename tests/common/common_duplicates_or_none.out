user: 12 spellbooks successfully added.

user: Success.

user: Duplicate students provided.

user: No such student in system.

user: Ending Archive process.
